app/helpers/Intersect.js

```diff
@@ -322,6 +322,30 @@ export function fillDayBins(
   return dayBins;
 }
 
+const transformDayBinsToExposureInfo = (dayBins) => {
```

Comment:
I know this interest.js file is quite large so it might not makes sense to convert the whole file, but what do you think about putting this guy into another ts file such that we can provide some type safety?

```typescript
const transformDayBinsToExposureInfo = (dayBins: DayBin[]): ExposureInfo => {
```

Reply:
I opened a ticket in backlog to refactor all of intersect.js and convert it to typescript, but it isn't gonna be pulled into this sprint. 

I can add the type declaration for this method to the `intersect.d.ts` if that will solve this problem for now.